core/src/main/java/nl/nn/adapterframework/metrics/FrankStatisticsRegistry.java

```diff
@@ -0,0 +1,43 @@
+/*
+   Copyright 2022 WeAreFrank!
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package nl.nn.adapterframework.metrics;
+
+import java.time.Duration;
+
+import io.micrometer.core.instrument.Clock;
+import io.micrometer.core.instrument.DistributionSummary;
+import io.micrometer.core.instrument.Meter.Id;
+import io.micrometer.core.instrument.distribution.DistributionStatisticConfig;
+import io.micrometer.core.instrument.distribution.HistogramGauges;
+import io.micrometer.core.instrument.simple.SimpleConfig;
+import io.micrometer.core.instrument.simple.SimpleMeterRegistry;
+
+public class FrankStatisticsRegistry extends SimpleMeterRegistry {
```

Comment:
Je moet alleen een nieuwe registry implementatie maken als je een nieuwe database zoals Prometheus of CloudWatch wil implementeren, niet als je DistributionSummaries wil tweaken.

Reply:
Je DistributionSummary doet dingen aan de meet-kant, niet aan de opslag-kant. Als je hem wil gebruiken, dan moet je hem niet in een speciale registry instantieren, maar daar waar je hem wil gebruiken, dus in de StatisticsKeeper.